clustered/integration-test/README.md

```diff
@@ -0,0 +1,8 @@
+How to launch tests from an IDE
+-------------------------------
+
+1. Create the kit with `gradle :clustered:integration-test:copyServerLibs`
+2. Launch the test once to make it fail
```

Comment:
Not as easy to debug (and on my IDE I can't launch gradle). But that could be another option. I will add it.